other/python_transformations/1_layer_on_substrate.py

```diff
@@ -17,38 +17,46 @@
 
 """BLOCK: Utils, Class Definitions, and main()"""
 """
-NOTE: edit the variables above for your specific use case.
+NOTE: edit the variables below for your specific use case.
 """
 
 # Indices identify the substrate and layer from the list of input materials under `materials_in` in globals().
 SUBSTRATE_INDEX = 0
 LAYER_INDEX = 1
 
 SETTINGS = {
-    "surface": {
-        # Set Miller indices (h, k, l) for the resulting substrate surface.
-        "miller:h": 1,
-        "miller:k": 1,
-        "miller:l": 1,
+    "substrate_surface": {
+        # Set Miller indices as a tuple for the resulting substrate surface.
+        "miller_indices": (1, 1, 1),
         # The vacuum space (in Ångströms) added to the surface in the direction perpendicular to the surface.
         "vacuum": 5,
         # The number of atomic layers in the resulting substrate.
         "number_of_layers": 3,
-    },
-    "interface": {
         # The transformation matrix for the surface. Format is: [[v1x, v1y], [v2x, v2y]].
-        "surface_v:matrix": [
+        "superlattice_matrix": [
             [1, 0],
             [0, 1]
         ],
-        # The transformation matrix for the layer. Format is the same as above.
-        "layer_v:matrix": [
+    },
+    "layer_surface": {
+        # Set Miller indices as a tuple for the resulting layer surface: (0,0,1) for 2D material
+        "miller_indices": (0, 0, 1),
+        # The vacuum space (in Ångströms) added to the surface in the direction perpendicular to the surface.
+        "vacuum": 5,
+        # The number of atomic layers in the resulting substrate: 1 for 2D material
+        "number_of_layers": 1,
+        # The transformation matrix for the surface. Format is: [[v1x, v1y], [v2x, v2y]].
+        "superlattice_matrix": [
```

Comment:
@chu-k How can we prevent Prettier from wrapping this

Reply:
Pre-commit hooks/black formatting shouldn't wrap this, pushed fixes